src/components/ApplicationDownloadLink.js

```diff
@@ -140,7 +155,20 @@ export default function ApplicationDownloadLink(props) {
   const internalCollaborators = getCollaborators(darInfo, 'internalCollaborators');
   const externalCollaborators = getCollaborators(darInfo, 'externalCollaborators');
   const labCollaborators = getCollaborators(darInfo, 'labCollaborators');
-
+  const nihUsernameProp = find({propertyKey: 'nihUsername'})(researcherProfile.researcherProperties);
```

Comment:
I'd love to see all string values like `'nihUsername'` extracted to constants in a util class. Might be a little beyond the scope of this PR, so maybe we can add a tech-debt story to address this. Another way to do this would be to extract all of these into a separate method that handles the extraction by taking in the properties and returning a full object. Something like `const researcherProps = utils.parseProfileProps(researcherProfile) { ... };`